apis/policies/v1alpha2/policyserver_types.go

```diff
@@ -79,10 +79,21 @@ const (
 	// PolicyServerServiceReconciled represents the condition of the
 	// Policy Server Service reconciliation
 	PolicyServerServiceReconciled PolicyConditionType = "PolicyServerServiceReconciled"
+	// Policy Server ready and active
+	PolicyServerServiceActive PolicyConditionType = "PolicyServerActive"
```

Comment:
The more I think about it, the more I become convinced that adding the status to the PolicyServer gives us nothing :/. 

Right now, we have 3 status:
- `unscheduled`: default, set on creation.
- `pending`:  if PolicyServer is currently being reconciled, or has failed to reconcile. 
-  `active`: PolicyServer has been reconciled.

Ideally, I would like to have the PolicyServer reflect the [status of their corresponding deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#deployment-status). This would give us "progressing, complete, failed", which makes more sense given that PolicyServers keep being ready and serving from the old pods, even if the new rolledout ones by the deployment never get ready.

@ereslibre already raised this point, as one needs a reconcile loop for the deployment, to be able to update the PolicyServer status when the pods or deployment change.

So, now I'm siding with just adding a label to the deployment, with the PolicyServer name (as we are doing in this PR already), but don't add nor implement any status on PolicyServer.

Sorry for the wasted efforts on this card :pensive: .